### PR TITLE
A bunch of task commit;  Replace ts() with E::ts(); Confirm civicrm coding standard;

### PR DESCRIPTION
**Pull request by lucky091588**

### CODE REVIEW

**Comment by lucky091588** [comment]
@twomice I already checked this and there is not `E` error.

File: autoincfield.php [hunk 1]
```diff
@@ -52,13 +50,14 @@ function autoincfield_civicrm_buildForm($formName, &$form) {
...
+      CRM_Core_Resources::singleton()->addScriptFile('com.joineryhq.autoincfield', 'js/CRM_Custom_Form_Field.js', 100, 'page-footer');
 
       // Create necessary fields
-      $form->addElement('checkbox', 'autoinc', ts('Is Autoincrement?'));
-      $form->addElement('text', 'min_value', ts('Minimum next value'));
+      $form->addElement('checkbox', 'autoinc', E::ts('Is Autoincrement?'));
+      $form->addElement('text', 'min_value', E::ts('Minimum next value'));
       // Assign bhfe fields to the template.
       $tpl = CRM_Core_Smarty::singleton();
+      $tpl = CRM_Core_Smarty::singleton();
```
Comment (twomice) [inline autoincfield.php]: @lucky091588 Looks like this line is redundant. Can you remove it from the PR? Thanks.

Comment (lucky091588) [reply autoincfield.php]: Done

**Comment by twomice** [comment]
Merged,  thanks @lucky091588

**Pull request merged**